installer/resources/pacbot_app/files/DB.sql

```diff
@@ -3119,7 +3119,9 @@ INSERT IGNORE INTO pac_v2_ui_options (filterId,optionName,optionValue,optionURL)
 INSERT IGNORE INTO pac_v2_ui_options (filterId,optionName,optionValue,optionURL) VALUES (19,'Type','groupType','/admin/asset-group/filter');
 INSERT IGNORE INTO pac_v2_ui_options (filterId,optionName,optionValue,optionURL) VALUES (19,'Number of assets','assetCount','/admin/asset-group/filter');
 INSERT IGNORE INTO pac_v2_ui_options (filterId,optionName,optionValue,optionURL) VALUES (19,'Created By','createdBy','/admin/asset-group/filter');
-
+INSERT IGNORE INTO pac_v2_ui_options (filterId,optionName,optionValue,optionURL) VALUES (17,'DisabledReason','DisabledReason','/compliance/v1/filters/policy');
```

Comment:
I believe display name for  DisabledReason, DisabledOn and DisableExpiredOn should not be in camel case

Reply:
@ershad-paladin as of now this requirement is PUT ON HOLD. So holding this PR